### DISCUSSION TITLE
Billing for Waiting for Approval wait period

### DISCUSSION THREAD

**Discussion by harshalpatel06** [community/community#59421] [Actions]
### Select Topic Area

General

### Body

Does github count the waiting time for waiting for approval jobs towards billing minutes ?

**Comment by jfang8** [comment]
yes. we found the problem.

it shows 6hours+ billable time when waiting for approval to deploy to PROD env.
VERY UNREASONABLE

**Reply by ratish11** [reply]
Hello is there any update on this issue? Are the github actions pipelines charged against waiting time.

**Reply by ratish11** [reply]
@jfang8  Can you please check
